src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  transition: all 300ms ease;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin: 40px 0 16px;
+  }
+
+  &__info-title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__info-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    margin: 4px 0 16px;
+  }
+
+  &__info-stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__info-reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__price-num {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__buy {
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    background-color: $color-accent-blue;
+    border-radius: 5px;
+    padding: 12px 70px;
+    transition: background-color 0.5s ease;
+  }
+
+  &__buy:hover {
+    border: 1px solid $color-accent-blue;
+    background-color: #fff;
+    color: $color-accent-blue;
+    padding: 11px 69px;
+  }
```

Comment:
one more simple comment, take advanantage of **scss** move `&__buy:hover` into body of `&__buy`

like this:
```
&__buy {
 // styles
 
 &:hover {
   // styles
 }
}
```

Reply:
done🫡 all changes added to the file